CRM/MembershipExtras/Upgrader.php

```diff
@@ -26,13 +28,69 @@ private function createOfflineAutoRenewalScheduledJob() {
       'api_action' => 'run',
     ]);
 
+
     $paymentProcessorType = new ManualRecurringPaymentProcessorType();
     $paymentProcessorType->create();
 
     $paymentProcessor = new OfflineRecurringPaymentProcessor();
     $paymentProcessor->create();
   }
 
+  /**
+   * Creates 'External ID' Custom Field for LineItem
+   */
+  private function createLineItemExternalIDCustomField(){
```

Comment:
There should be a space between ')' and '{', ie. `private function functionname() {`.